cleanlab/datalab/internal/issue_finder.py

```diff
@@ -72,6 +76,7 @@ class IssueFinder:
 
     def __init__(self, datalab: "Datalab", verbosity=1):
         self.datalab = datalab
+        self.task = self.datalab.task
```

Comment:
Can we just inject the task into this constructor, instead of increasing the coupling between Datalab and IssueFinder?

```suggestion
    def __init__(self, datalab: "Datalab", task:str, verbosity=1):
        self.datalab = datalab
        self.task = task
```

Update docstring accordingly!

Reply:
Hi @elisno,

I'm not entirely certain about this suggestion. For example, I'm unsure about how IssueFinder should behave if I initialize it with a Datalab that has the task set to classification and then provide the task "regression" in a separate argument as you suggest. Currently, issueFinder obtains task from datalab.

Reply:
I don't think we should allow changing the task type in a constructed Datalab. It should be fixed forever IMO